disnake/state.py

```diff
@@ -2097,9 +2097,7 @@ async def _delay_ready(self) -> None:
         self._ready_task = None
 
         # update member references once guilds are chunked
-        # (if we're not chunking guilds, there's no use in trying to update member references)
-        if self._chunk_guilds:
-            self._update_member_references()
```

Comment:
should leave a comment here as to why its always rebound, instead of just the commit message

Reply:
yup, good idea
https://github.com/DisnakeDev/disnake/pull/425/commits/f9b118d57bc43409c4cb5ce3a008a8a5137e8d5b